clientLibrary/src/test/scala/za/co/absa/loginclient/authorization/ClaimsParserTest.scala

```diff
@@ -80,23 +81,23 @@ class AccessClaimsParserTest extends AnyFlatSpec with Matchers {
   }
 
   it should "return the email address" in {
-    val email = AccessTokenClaimsParser.getEmail(accessJwt)
-    assert(email.get.equals(FakeTokens.email))
+    val email = AccessTokenClaimsParser.getOptionalClaims(accessJwt)
+    assert(email("email").toString.equals(FakeTokens.email))
   }
 
   it should "return an empty email if email claim is not present at all" in {
-    val email = AccessTokenClaimsParser.getEmail(missingAllClaimsButSubjectJwt) // does not throw
-    email shouldBe None
+    val email = AccessTokenClaimsParser.getOptionalClaims(missingAllClaimsButSubjectJwt) // does not throw
+    email shouldBe Map.empty
   }
 
   it should "return the display name" in {
-    val displayName = AccessTokenClaimsParser.getDisplayName(accessJwt)
-    assert(displayName.get.equals(FakeTokens.displayName))
+    val displayName = AccessTokenClaimsParser.getOptionalClaims(accessJwt)
+    assert(displayName("displayname").toString.equals(FakeTokens.displayName))
   }
 
   it should "return an empty display name if displayname claim is not present at all" in {
-    val dn = AccessTokenClaimsParser.getDisplayName(missingAllClaimsButSubjectJwt) // does not throw
-    dn shouldBe None
+    val dn = AccessTokenClaimsParser.getOptionalClaims(missingAllClaimsButSubjectJwt) // does not throw
+    dn shouldBe Map.empty
   }
```

Comment:
These test cases are a bit weird.

`val email = AccessTokenClaimsParser.getOptionalClaims(accessJwt)` does not make sense (it is the optional claims map that get returned


Perhaps you could just either:
(a) merge these test cases for situations where
 - optional claims are present
 - optional claims are not present
  (because now it is the same optional claims map that gets returned)
or
(b) use `getClaim(jwt: Jwt, claimKey: String)` for a specific claim existence test in the current test cases structure. 

Or both somehow.